src/main/java/io/jenkins/plugins/ml/IPythonKernelInterpreter.java

```diff
@@ -89,6 +91,19 @@ public InterpreterResultMessage interpretCode(String code) throws IOException, I
         return interpreterResultMessage;
     }
 
+    public List<InterpreterResultMessage> interpretComplexCode(String code) throws IOException, InterpreterException {
+        InterpreterResult result;
+        InterpreterContext context = getInterpreterContext();
+        result = interpreter.interpret(code, context);
+        LOGGER.info(result.code().toString());
+        List<InterpreterResultMessage> rst = context.out.toInterpreterResultMessage();
+
+        if (rst.size() > 0) {
+            return rst;
+        }
+        return null;
+    }
```

Comment:
@kinow we can probably delete `interpretCode` method and rename `interpretComplexCode` as `interpretCode` because we need the list of outputs not a single message not so use full in future.